README.md

```diff
@@ -33,6 +32,9 @@ doctor.collect(['node', './path-to-script.js'], function (err, filepath) {
 })
 ```
 
+You can find examples in
+[node-clinic-doctor-examples](https://github.com/nearform/node-clinic-doctor-examples/blob/master/slow-io/index.js#L29)
```

Comment:
Shouldn't this be pointing to the full repo?

Reply:
Yes, likely. I'll change that as well.

Reply:
Addressed